styles.css

```diff
@@ -0,0 +1,105 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: inter, Arial, Verdana, Tahoma, sans-serif;
+  list-style: none;
+}
+
+@media only screen and (max-width: 600px) {
+  .navbar {
+    display: flex;
+    justify-content: space-between;
+    width: 100vw;
+    height: 3rem;
+    padding: 1rem;
+  }
+
+  .navbar .shortcuts {
+    display: none;
+  }
+
+  .navbar h2 {
+    font-size: 1.25rem;
+    font-weight: 600;
+    line-height: 1.75rem;
+  }
+
+  .options {
+    width: 1.5rem;
+    height: 1.5rem;
+  }
+
+  .social i {
+    font-size: 2rem;
+    color: rgb(59, 59, 59);
+  }
+
+  .options i {
+    font-size: 1.5rem;
+  }
+
+  .social i:hover {
+    color: rgb(45, 204, 151);
+  }
+
+  .options i:hover {
+    color: rgb(45, 204, 151);
+  }
+
+  #homepage {
+    display: flex;
+    max-width: 100vw;
+    min-height: 100vh;
+    background-image: url(header-illsutration-mobile@2x.png);
+    background-size: 100vw;
+    background-repeat: no-repeat;
+    border-bottom-left-radius: 2rem;
+    flex-direction: column;
+    padding: 2rem;
+    justify-content: space-around;
+    align-items: center;
+  }
+
+  #homepage h1 {
+    color: rgb(0, 0, 88);
+    font-size: 2.5rem;
+    text-align: center;
+    font-weight: 700;
+  }
+
+  #homepage h1 span {
+    color: rgb(45, 204, 151);
```

Comment:
[OPTIONAL] 
Please stick to the design and follow color code on the Figma templete.
<img width="564" alt="Screenshot 2023-01-12 at 11 51 00 PM" src="https://user-images.githubusercontent.com/25887253/212178930-eb979e90-fe82-4177-99c3-ae2df0021ee5.png">

Reply:
Okay, I'll do it, thanks for the feedback! :D